spec/requests/facebook_omniauth_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Omniauth', type: :request do
+  describe 'GET /auth/facebook/callback' do
+    context 'when data is valid' do
+      before(:each) do
+        valid_facebook_login_setup
+        get '/users/auth/facebook/callback'
+        request.env['omniauth.auth'] = OmniAuth.config.mock_auth[:facebook]
+      end
+
+      it 'should set user id' do
```

Comment:
don't use `should` word

http://rspec.info/blog/2012/06/rspecs-new-expectation-syntax/